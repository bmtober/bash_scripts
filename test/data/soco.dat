alice
bob
bob
cindy
bob
alice
cindy
david
david
david
elaine
bob
alice
david
alice
david
david
elaine
bob
david
