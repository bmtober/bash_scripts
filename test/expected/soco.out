      7 david
      5 bob
      4 alice
      2 elaine
      2 cindy
